_layouts/personas.html

```diff
@@ -34,7 +34,8 @@
                     <img src="{{ site.baseurl }}/assets/personas/mugshots/{{ page.img }}" alt="Headshot of {{ page.title }}">
                     <h1>{{ page.title }}</h1>
                     {% endif %}
-                    {{ content }}
+                    <!--{{ content }}-->
```

Comment:
@mgifford I think it is better to just delete this line then generate two versions of the content in the source which this might do.

Reply:
I've modified this, and agree. I just wanted to to express exactly what i'd written there as a comment, not spit out the results of the comment. But not sure how to do that.